package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mofo-style",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
this should probably be a new major (2.0.0) rather than just minor; it's absolutely backward compat. breaking =D